core/build.sbt

```diff
@@ -15,6 +15,8 @@ libraryDependencies ++= Seq(
 
 scalacOptions += "-Ypartial-unification"
 
+Test / testOptions += Tests.Argument("tmpdir", (target.value / "tests" / "tmp").getPath)
```

Comment:
On `target` so they get cleaned with `clean`. Not auto-cleaned so that I can look at the files if I need to debug the test. I did forget to delete it on test start. 

Also of note: stable path, not random, so one can easily find it.